index.html

```diff
@@ -22,7 +22,16 @@
     <div id='page'>
 <h1>Welcome to the Tamarin Manual</h1>
 <p class="halfbreak">
-The manual is available as a <a href='tex/tamarin-manual.pdf'>PDF</a> or as <a href='book/001_introduction.html'>HTML</a> for online browsing.
+The manual is available as a
+<a href='master/tex/tamarin-manual.pdf'>PDF</a>
+or as 
+<a href='master/book/001_introduction.html'>HTML</a>
+for online browsing. 
+(For developers, see here for the 
```

Comment:
Maybe make a bigger visual distinction would be better. Maybe we should also add/modify a robots.txt so that, e.g., google only references the master version. Otherwise we might have questions from people that found features on Google, but which are not yet in the release version.

Reply:
I agree with that but I know to little HTML to do that.

Reply:
Great!

Reply:
No problem, I can add a robots.txt to tamarin-prover.github.io

Reply:
great!